trailblazer/clients/tower/utils.py

```diff
@@ -0,0 +1,27 @@
+import logging
+from functools import wraps
+from pydantic import ValidationError
+from requests import ConnectionError, HTTPError
+from requests.exceptions import MissingSchema
+
+from trailblazer.exc import InvalidTowerAPIResponse, TowerAPIClientError, TowerRequestFailed
+
+LOG = logging.getLogger(__name__)
+
+
+def handle_client_errors(func):
```

Comment:
Add docstring and type hint

Reply:
Added docstring here https://github.com/Clinical-Genomics/trailblazer/pull/463/commits/e688385e7f245d3a6786b0e9fa5a5d36c91f4f87

The type hint for the function would not add any clarity and I will omit it